iib/workers/tasks/utils.py

```diff
@@ -1191,19 +1185,27 @@ def prepare_request_for_build(
 
     binary_image = build_request_config.binary_image(request_index, distribution_scope)
     if binary_image == "scratch":  # binaryless image mode
+        arches = set(["linux-amd64"])
```

Comment:
Is the architecture correct, does it pass? 
I cant find any valid list of architectures, just this:


> Set the ARCH of the image to be built, and that of the base image to be pulled, if the build uses one, to the provided value instead of using the architecture of the host. (Examples: arm, arm64, 386, amd64, ppc64le, s390x)


https://github.com/containers/buildah/blob/main/docs/buildah-build.1.md

Reply:
OK, for our purposes I got this - we deliver just this list: 
```bash
podman manifest inspect registry.redhat.io/redhat/redhat-operator-index:v4.16 | grep architecture
                "architecture": "amd64",
                "architecture": "arm64",
                "architecture": "ppc64le",
                "architecture": "s390x",
```

Could not w use just the `amd64` instad of `linux-amd64` ?

Reply:
Sure, I'll change it to `arm64` instead

Reply:
Patched